openapi.yaml

```diff
@@ -7091,7 +7053,18 @@ components:
         - callbacks
     workspace:
       type: object
+      xml:
+        name: workspace
+        prefix: ns3
+      x-tags:
+        - Workspaces
+      title: Workspace
       properties:
+        id:
```

Comment:
Updated 'Id' attribute part of workspace model to include examples and a 'read-only' property